src/Files.App/Views/Settings/DevToolsPage.xaml

```diff
@@ -11,6 +11,8 @@
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	xmlns:uc="using:Files.App.UserControls"
 	xmlns:vm="using:Files.App.ViewModels.Settings"
+	xmlns:wctconverters="using:CommunityToolkit.WinUI.Converters"
+	SizeChanged="Page_SizeChanged"
```

Comment:
Can we use the built-in triggers for screen size?

Reply:
> If you mean something like this, it works!

Yes

> is there a predefined width we should use as a breakpoint?

It should be able to fit the text box and browse button, with a little extra space for other localizations where the browse text may be longer.

Reply:
This is the min size in the latest commit
![image](https://github.com/user-attachments/assets/81c6d7ae-9388-4ab0-b6c4-8369b49fe876)

Reply:
BTW I used two `AdaptiveTriggers` to prevent a bug where the `Compact` layout was not triggered